src/librustc_unicode/char.rs

```diff
@@ -362,7 +362,7 @@ impl char {
     /// mostly similar to ID_Start but modified for closure under NFKx.
     #[unstable(feature = "unicode",
                reason = "mainly needed for compiler internals",
-               issue = "0")]
+               issue = "27784")]
```

Comment:
These actually aren't quite related to the character encoding methods because they're basically just compiler internals at this point, I don't think these are on the track to stabilization any time soon at all.